Playground/js/main.js

```diff
@@ -1073,6 +1073,11 @@ class Main {
             }
         }
         if (pgHash) {
+            var match = pgHash.match(/^(#[A-Z\d]*)(%23)([\d]+)$/);

```

Comment:
If you can please add [a-z] to the PG-matcher as well (right next to the [A-Z], just to stay future safe

Reply:
done